usaspending_api/etl/subaward_etl.py

```diff
@@ -12,7 +12,7 @@
 
 # These queries are directly from the broker
 D1_FILE_F_QUERY = """
-SELECT *
+SELECT DISTINCT ON (subcontract_num, fsrs_procurement.contract_number, fsrs_procurement.idv_reference_number) *
```

Comment:
Nice-- `DISTINCT ON` is on of my favorite Postgres features. Another subaward n00b question...does this mean that our source data has duplicate records and we are trying to filter those out? Are we certain we can throw away the extras?